src/subcommand/wallet/sell.rs

```diff
@@ -0,0 +1,78 @@
+use {
+  super::*,
+  bitcoin::{
+    blockdata::{locktime::absolute::LockTime, witness::Witness},
+    psbt::Psbt,
+    sighash::{EcdsaSighashType, TapSighashType},
+  },
+  bitcoincore_rpc::json::SigHashType,
+};
+
+#[derive(Debug, Parser, Clone)]
+pub(crate) struct Sell {
+  #[arg(
+    long,
+    help = "The amount of sats to sell your inscription or rare sats for."
+  )]
+  pub amount: Amount,
+  pub outgoing: Outgoing,
```

Comment:
Might be nice if you could optionally specify a particular address for the sales funds to go to.